third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -212,6 +212,8 @@ The following arguments are supported:
 
 * `replica_configuration` - (Optional) The configuration for replication. The
     configuration is detailed below.
+    
+* `root_password` - (Optional) Required for MS SQL Server, ignored MySQL and PostgreSQL.
```

Comment:
Can you also mention in `database_version` what the accepted MSSQL versions are?

Reply:
sure, I'll take those from discovery document

Reply:
done